src/main/java/frc/robot/commands/ArcadeDriveSlowMode.java

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Constants;
+import frc.robot.LogitechGamingPad;
+import frc.robot.subsystems.DriveTrain;
+
+public class ArcadeDriveSlowMode extends CommandBase {
+  /** Creates a new ArcadeDriveSlowMode. */
+  DriveTrain driveTrain;
+  LogitechGamingPad drivePad;
```

Comment:
nit: `private final`
```suggestion
  private final DriveTrain driveTrain;
  private final LogitechGamingPad drivePad;
```